app/jobs/apply_api/import_applications_job.rb

```diff
@@ -9,7 +9,9 @@ def perform
 
       new_applications.each do |application_data|
         application_record = ImportApplication.call(application_data: application_data)
-        Trainees::CreateFromApply.call(application: application_record)
+        Trainees::CreateFromApply.call(application: application_record) if application_record.present?
+      rescue ApplyApi::ImportApplication::ApplyApiMissingDataError
+        Sentry.capture_message "The data from Apply has missing attributes, Application ID: #{application_data['id']}"
```

Comment:
Can we use `Sentry.capture_exception(exception)` here instead so as we get the stack trace? We could add this message to the exception when it gets raised so the extra info still gets passed.